platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/LogoutRequestService.java

```diff
@@ -383,6 +386,12 @@ private void logout() {
 
         SecurityTokenHolder tokenHolder = ((SecurityTokenHolder) session.getAttribute(
                 SecurityConstants.SAML_ASSERTION));
+
+        org.apache.shiro.subject.Subject subject = ThreadContext.getSubject();
+        Arrays.stream(System.getProperty("security.audit.roles").split(",")).filter(
```

Comment:
Same comments as before re: formatting, `splitAsStream`, and method reference. 

I was going to suggest creating a method in `SubjectUtils` but there isn't all that much here that would make extracting it worthwhile.